packages/nextjs/src/server/index.ts

```diff
@@ -1,3 +1,7 @@
-export * from './clerk';
+export * from './clerkClient';
 export * from './getAuth';
 export * from './withClerkMiddleware';
+
+export { auth } from '../app-router/server/auth';
+export { currentUser } from '../app-router/server/currentUser';
+export { authMiddleware } from './authMiddleware';
```

Comment:
❓ should we mark the `getAuth`, `withClerkMiddleware` as deprecated in a following PR (when we consider the new API stable) ?

Reply:
`getAuth` is not being deprecated for now, we still need it for `/pages`